gtda/mapper/visualization.py

```diff
@@ -232,6 +233,10 @@ def plot_interactive_mapper_graph(pipeline, data, layout='kamada_kawai',
     plotly_kwargs : dict, optional, default: ``None``
         Keyword arguments to configure the Plotly Figure.
 
+    in_place : bool, optional, default: ``False``
```

Comment:
Another option would be to call this parameter `clone` (with default ``True``). Not sure what would be more standard, will try to look around.

Reply:
Note also that `plot_static_mapper_graph` already has a `clone_pipeline` parameter with the same meaning.